src/SurveyService.Business/Commands/Question/CreateQuestionCommand.cs

```diff
@@ -19,14 +19,13 @@ public class CreateQuestionCommand : ICreateQuestionCommand
   private readonly IHttpContextAccessor _httpContextAccessor;
   private readonly IDbSingleQuestionMapper _mapper;
   private readonly IQuestionRepository _questionRepository;
-  private readonly ICreateQuestionRequestValidator _validator;
-  private readonly IResponseCreator _responseCreator;
+  private readonly ICreateSingleQuestionRequestValidator _validator;
 
```

Comment:
you must have accidentally removed _responseCreator

Reply:
DOne